lambdas/service/app.py

```diff
@@ -1171,25 +1144,31 @@ def repository_search(entity_type: str,
     description='The name of the catalog to query.')
 
 
-def repository_search_params_spec(index_name):
-    sort_default, order_default = sort_defaults[index_name]
+def repository_search_params_spec():
+    catalog_spec_link = '#operations-Index-get_index_catalogs'
     return [
         catalog_param_spec,
         filters_param_spec,
+        params.path(
+            'entity_type',
+            str,
+            description=f'Which index to search. The set of available indices '
```

Comment:
Don't expose the notion of multiple indexes to the user. That we maintain an index per entity type should remain an implementation detail.

Reply:
So this could say "The type of the entities to be returned" or sth like that.

Reply:
I'm happy to make this change, but I will point out that the on `develop` the documentation currently includes the following language:

> Azul denormalizes and aggregates metadata into **several different indices** for selected entity types ... This **set of indexes** forms a catalog.

Should I edit this section as well?

Reply:
Decided at parking lot to revert to the original language for consistency with the existing documentation.